tests/integration/base_block_test.go

```diff
@@ -211,7 +198,19 @@ metadata:
 provisioner: rook.io/block
 parameters:
     pool: ` + poolName + `
-    clusterName: ` + namespace
+    clusterName: ` + namespace +`
```

Comment:
combining the pvc will work for the tests? I thought it would still potentially cause an error state before succeeding. The tests are ok with this temporary error?

Reply:
Tests should pass even with a potential error state, Test waits until PVC is bound before proceeding to mount.